playbooks/files/backup/backup.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env -S sh -e -u
+#   Copyright (C) 2024-2025 Martina Ferrari, Simó Albert i Beltran, Adeodato Simó
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+BACKUP_FROM_DIR="${BACKUP_FROM_DIR:-/tmp/bk-snap}"
+
+clean_mount_point(){
+  # Umount if mounted and always remove the directory
+  local mount_point_dir=$1
+  if mountpoint -q $mount_point_dir
+  then
+    umount $mount_point_dir
+  fi
+  # Remove mount point directory but maybe nothing is mounted due to an interruption
+  if test -d $mount_point_dir
+  then
+    rmdir $mount_point_dir
+  fi
+}
+
+get_backup_location(){
+  # Get backup location for device name
+  local device_name=$1
+  echo "$BACKUP_FROM_DIR/$(dmsetup info -C --noheadings -o name $device_name | sed 's/--snap$//')"
```

Comment:
here and in other places, it'd be good to wrap at 80 chars

Reply:
I will do it.

Reply:
I done it via https://github.com/Tina-pm/ansible-playbooks/pull/20/commits/828c98ae80d8b8c2bb38d290d2ddb75841e3323a

Reply:
I am going to resolve it due to the approval https://github.com/Tina-pm/ansible-playbooks/pull/20#pullrequestreview-2573063558